### PR TITLE
chore: add logo

### PR DESCRIPTION
**Pull request by vadasambar**
## Considerations
- Something that incorporates the image of water (aqua) and package
- Minimalistic: should express the idea with less design components
- Scalable: should look good irrespective of the scale (favicon, t-shirts, print-outs and everything else)
- Built with open source: 
   - Built with [Inkscape](https://inkscape.org/) 0.91
   - Font: [Swasdee Regular](https://en.fontke.com/font/12135927/) ([License](https://en.fontke.com/font/12135927/licence/))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# aqua
+<p align="center" width="100%">
+<img src="./logo/aqua_horizontal.svg" width="400">
```
Comment (vadasambar) [inline README.md:2]: You can check how it looks in the README [here.](https://github.com/suzuki-shunsuke/aqua/blob/3440f684fc9e782ea5adf2f2d4ad613fa0657f45/README.md)

File: logo/aqua_vertical.svg [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (vadasambar) [inline logo/aqua_vertical.svg:1]: SVG files are stored as XML

**Comment by vadasambar** [comment]
@suzuki-shunsuke, can you please review this when you find free time :pray: 
Thank you!

**Review by suzuki-shunsuke — APPROVED** [review]
Thank you for your contribution!
The logo is very cool. I love it.

**Comment by suzuki-shunsuke** [comment]
Added the description. https://github.com/suzuki-shunsuke/aqua/pull/405

**Pull request merged**